webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DataConverter.scala

```diff
@@ -47,14 +47,16 @@ trait DataConverter extends FoxImplicits {
     dstArray
   }
 
-  def toUnsigned(data: Array[_ >: Byte with Short with Int with Long with Float])
-    : Array[_ >: UByte with UShort with UInt with ULong with Float] =
+  def toMaybeUnsigned(
```

Comment:
@fm3 regarding naming here: In the backend the word `opt` is usually used, correct? But in this case, no real Option is involved. So is it ok to keep this name? Do you have better suggestions?

Reply:
how about `toUnsignedIfNeeded`? I think we have “ifNeeded” already in a couple of places.